### ISSUE TITLE
Youtube V3 Data API - Channels.list function not fetching all the channels

### ISSUE THREAD

**Issue by dpramani** [googleapis/google-api-nodejs-client#2613]
Hi, I am trying to get a list of youtube channels under one google account.  But the api returns only one channel, although I have two channels inside my google account.  Is there a way to get all the channels under one google account ? Is there any suggested workaround to this ? I have seen posts saying this is not possible, is this same now ? 

Thanks

`const service = google.youtube({
      version: 'v3',
      headers: {
        Authorization: `Bearer ${user.youtubeToken}`,
      },
    });

    const result = service.channels.list({
      part: 'snippet,contentDetails,statistics',
      mine: true,
    });
`

**Comment by tmatsuo** [comment]
@dpramani Where are the posts you found?

**Comment by tmatsuo** [comment]
Did you try `managedByMe: true`?

**Comment by dpramani** [comment]
@tmatsuo Yes I tried managedByMe: true, but got an error saying : 'Error: Access forbidden. The request may not be properly authorized.' 

Some similar posts which I found are:
https://issuetracker.google.com/issues/35175143
https://stackoverflow.com/questions/40781463/how-i-retrieve-the-chanel-list-using-youtube-api

**Comment by bhavikfreelanace9085** [comment]
did you find the solution???

**Comment by david1asher** [comment]
have same issue, anyone found solution?

**Comment by 0dayplug** [comment]
yes definitely! getting a 403 access forbidden error. Any solutions or work around for this please ?
```
{'message': 'Access forbidden. The request may not be properly authorized.', 'domain': 'youtube.common', 'reason': 'forbidden'}
```

**Comment by Bogatinovski** [comment]
I am also getting the same issue, only receiving my own YouTube account and not the others on which I am assigned as an Admin. I tried adding the `managedByMe` parameter and it gives "Forbidden" error.
